app/experimenter/static/js/components/Branch.js

```diff
@@ -17,28 +17,27 @@ class Branch extends React.PureComponent {
 
   handleChange(key, value) {
     const { onChange, branch } = this.props;
-    const updated = branch
-      .set("is_control", this.props.index === 0)
```

Comment:
Noticed I forgot this in here after I did the control stuff, so got rid of that.